src/coldfront_plugin_api/tests/unit/test_users.py

```diff
@@ -82,3 +83,11 @@ def test_create_user_minimal_fetched(self):
         self.assertEqual(user_dict["name"]["givenName"], "fake")
         self.assertEqual(user_dict["name"]["familyName"], "user 1")
         self.assertEqual(user_dict["emails"][0]["value"], "fake_user_1@example.com")
+
+    def test_reseponse_404(self):
+        # Make a http request to scim endpoint
```

Comment:
This comment is not necessary and quite far from the line of code that it applies to, please remove it.